temporal-spring-boot-autoconfigure-alpha/src/main/java/io/temporal/spring/boot/autoconfigure/RootNamespaceAutoConfiguration.java

```diff
@@ -86,7 +88,9 @@ public NamespaceTemplate rootNamespaceTemplate(
       @Autowired(required = false) @Nullable
           TemporalOptionsCustomizer<WorkerOptions.Builder> workerCustomizer,
       @Autowired(required = false) @Nullable
-          TemporalOptionsCustomizer<WorkflowClientOptions.Builder> clientCustomizer) {
+          TemporalOptionsCustomizer<WorkflowClientOptions.Builder> clientCustomizer,
+      @Autowired(required = false) @Nullable
+          TemporalOptionsCustomizer<ScheduleClientOptions.Builder> scheduleCustomize) {
```

Comment:
Question, can you or your tests confirm that, due to type erasure, these last two parameters are properly treated as separate things by Spring?

Reply:
agree we need customization tests for this. will try to add